.github/workflows/build-and-deploy.yml

```diff
@@ -70,6 +71,7 @@ jobs:
           path: origin-bucket-metadata.json
 
       - name: Archive Sentry release
+        if: ${{ github.repository == 'pulumi/docs' }}
```

Comment:
```suggestion
```

I think we can just leave this line out, since there's no real reason _not_ to create build artifacts for these files but continue to do so for the others.

Reply:
This will be needed if we are not generating a Sentry artifact, then there is no artifact to publish. This step failed for me when I didn't have a Sentry token.

Reply:
To add to what I said above, if we choose to go down the path of having a private mirror, whenever we decide that, we can update this to include that repo. Since we will want to be deliberate about using the same Sentry token elsewhere.